### PR TITLE
Reserve "Travel Discovery" for #4620

### PR DESCRIPTION
**Pull request by tinahsieh**
The string changes are for fixing bug #4620. There're 5 new strings added including the non-translatable string "Travel Discovery".

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -958,29 +958,37 @@
...
+    <string name="travel_discovery" translatable="false">Travel Discovery</string>
+
+
     <!--Dialog-->
     <!--dialog title for introducing Firefox Lite Travel Discovery-->
     <string name="travel_dialog_1_title">Discover More!</string>
     <!--dialog description for introducing the feature. The %1$s here is Firefox Lite.-->
-    <string name="travel_dialog_1_description">Explore things to do, accommodations, and more in %1$s Travel Discovery.</string>
-    <!--Action button for trying Firefox Lite's new feature "Travel Discovery"-->
+    <string name="travel_dialog_1_description">Explore things to do, accomodations, and more in %1$s Travel Discovery.</string>
```
Comment (Conjuror) [inline app/src/main/res/values/strings.xml]: spelling error: accommodation

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -958,29 +958,37 @@
...
+    <!--dialog description for introducing the feature. the first %1$s is Firefox Lite, and the %2$s is Travel Discovery.-->
+    <string name="travel_dialog_1_description_new">Explore things to do, accomodations, and more in %1$s %2$s.</string>
```
Comment (Conjuror) [inline app/src/main/res/values/strings.xml]: spelling error: accommodation

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -958,29 +958,37 @@
     <string name="travel_find_destination_placeholder_2">Enter a destination</string>
     <!--this is a title indicates that users will see search results from google after tapping on the following option. the %1$s here is google. -->
     <string name="travel_search_engine_1">See results in %1$s</string>
-    <!--this is a title indicates that users will see search results summerized by smart travel search after tapping on the following options. -->
-    <string name="travel_search_engine_2">See results in smart travel search</string>
```
Comment (Conjuror) [inline app/src/main/res/values/strings.xml]: missing string

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -958,29 +958,37 @@
...
     <string name="travel_dialog_2_title">Enjoy Using Travel Discovery?</string>
+    <!--dialog title for asking if users want to change default search engine. the %1$s is Travel Discovery.-->
+    <string name="travel_dialog_2_title_new">Enjoy Using %1$s?</string>
     <!--dialog description for introducing how it works. The %1$s here is Firefox Lite.-->
     <string name="travel_dialog_2_description">You can set %1$s Travel Disovery as your default search engine for travel.</string>
     <!--checkbox for not showing this dialog again-->
     <string name="travel_dialog_2_checkbox">Do not ask me again</string>
     <!--Action button for setting Travel Discovery as default search engine-->
-    <string name="travel_dialog_2_action">SET AS DEFAULT</string>
+    <string name="travel_dialog_2_action">SET AS DEFAULT</string>
```
Comment (benjamin-cheng) [inline app/src/main/res/values/strings.xml]: Please help to remove the extra space you added in the commit.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (benjamin-cheng) [reply app/src/main/res/values/strings.xml]: I've added the commit to remove this unused string from all languages.

**Pull request merged**